tests/pyfunc/docker/test_docker.py

```diff
@@ -94,17 +95,18 @@ def _build_image_with_copy(context_dir, image_name):
         content = dockerfile.read_text()
         content = content.replace(VERSION, get_released_mlflow_version())
         dockerfile.write_text(content)
-
         shutil.copytree(context_dir, dst_dir)
-        for _ in range(3):
-            try:
-                # Docker image build is unstable on GitHub Actions, retry up to 3 times
-                build_image_from_context(context_dir, image_name)
-                break
-            except RuntimeError:
-                pass
-        else:
-            raise RuntimeError("Docker image build failed.")
+        # Build the image if the slow--tests flag is enabled
+        if _MLFLOW_RUN_SLOW_TESTS.get():
```

Comment:
this test is one of the most time consuming tests:

```
151.10s call     tests/pyfunc/docker/test_docker.py::test_build_image[params3]
148.82s call     tests/pyfunc/test_pyfunc_model_with_type_hints.py::test_predict_model_with_type_hints
139.69s call     tests/pyfunc/docker/test_docker.py::test_build_image[params7]
```

by skipping the image build, we can save 5~6 minutes.